composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "facebook/php-business-sdk",
+  "name": "oraybeheer/facebook-php-business-sdk",
```

Comment:
Please remove this file as this file should not be part of the PR. I believe you have added this to your testing needs.